frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -13,6 +13,7 @@ local T = require("ffi/util").template
 local ReaderDictionary = InputContainer:new{
     data_dir = nil,
     dict_window_list = {},
+    lookup_msg = _("Dictionary lookup of :\n“%1” ...")
```

Comment:
Yeah, I'd probably say something like "(looking up|searching for) (?:definition of)"